lib/widgets/matrix.dart

```diff
@@ -78,6 +83,7 @@ class MatrixState extends State<Matrix>
   String? loginUsername;
   LoginType? loginType;
   bool? loginRegistrationSupported;
+  bool? _twakeSupported;
```

Comment:
because you have method `setTwakeSupported` and also getter, so make it public instead of private